libvalkey/test.sh

```diff
@@ -1,43 +1,43 @@
 #!/bin/sh -ue
 
-REDIS_SERVER=${REDIS_SERVER:-redis-server}
-REDIS_PORT=${REDIS_PORT:-56379}
-REDIS_SSL_PORT=${REDIS_SSL_PORT:-56443}
+VALKEY_SERVER=${VALKEY_SERVER:-valkey-server}
+VALKEY_PORT=${VALKEY_PORT:-56379}
+VALKEY_SSL_PORT=${VALKEY_SSL_PORT:-56443}
 TEST_SSL=${TEST_SSL:-0}
 SKIPS_AS_FAILS=${SKIPS_AS_FAILS:-0}
 ENABLE_DEBUG_CMD=
 SSL_TEST_ARGS=
 SKIPS_ARG=${SKIPS_ARG:-}
-REDIS_DOCKER=${REDIS_DOCKER:-}
+VALKEY_DOCKER=${VALKEY_DOCKER:-}
 
-# We need to enable the DEBUG command for redis-server >= 7.0.0
-REDIS_MAJOR_VERSION="$(redis-server --version|awk -F'[^0-9]+' '{ print $2 }')"
-if [ "$REDIS_MAJOR_VERSION" -gt "6" ]; then
+# We need to enable the DEBUG command for valkey-server >= 7.0.0
```

Comment:
There is no Valkey < 7.2.5, so we should always add `enable-debug-command local`

Reply:
not sure, but maybe we want to be able to test with redis here, then we could use `$VALKEY_SERVER --version...` to get the version instead

Reply:
That could be a separate PR, just mentioning..